robozonky-strategy-natural/src/main/antlr4/imports/CommonFilters.g4

```diff
@@ -82,17 +81,17 @@ termCondition returns [MarketplaceFilterCondition result]:
 
 termConditionRangeOpen returns [MarketplaceFilterCondition result]:
     IS min=intExpr UP_TO max=intExpr
-    { $result = new LoanTermCondition($min.result, $max.result); }
+    { $result = LoanTermCondition.exact($min.result, $max.result); }
 ;
 
 termConditionRangeClosedLeft returns [MarketplaceFilterCondition result]:
     MORE_THAN min=intExpr
-    { $result = new LoanTermCondition($min.result + 1); }
+    { $result = LoanTermCondition.moreThan($min.result); }
 ;
 
 termConditionRangeClosedRight returns [MarketplaceFilterCondition result]:
     LESS_THAN max=intExpr
-    { $result = new LoanTermCondition(0, $max.result - 1); }
```

Comment:
I think it's these `+ - 1`s that make some of the existing integration tests for the web tool fail. For example previous parser accepted condition like this:
`Ignorovat vše, kde: délka nedosahuje 85 měsíců.`

Anyway after tweaking those +1s in the generator, the tests are passing fine. If you're ok with that I'll push the changes in my integration tests.

Reply:
This change is intentional - before this change, the conditions represented ranges closed on both sides. (`<A, B>`) To use an example, a way to make sure a range doesn't accept a number `B` was to specify it as `<A, B - 1>`.

With the new code, ranges are now either "less than A" `(-inf; A)`, "more than B" `(B; +inf)` or "anything between C and D" `<C; D>`. So, the behavior you are seeing is the expected behavior.

Feel free to merge your changes, I'll merge this PR. Thanks for checking, @jhrcek !